sdk/mapper.js

```diff
@@ -66,8 +66,12 @@ class Mapper {
         }
       }
     };
-    if (responseApi.hasReturn()) {
-      payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
+    try {
+      if (responseApi.hasReturn()) {
+        payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
```

Comment:
Right, but you're assigning `responseApi.getReturnType()` to that property. That's the **type** of data that can be returned, not the actual **value** itself, which it should be (and is only set if a value is actually returned).